.github/workflows/test.yml

```diff
@@ -0,0 +1,39 @@
+name: test
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  test-evaluation-js:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: 'evaluation/js'
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 'lts/Iron'
+          cache: 'npm'
+          cache-dependency-path: '**/**/package-lock.json'
+      - run: npm ci
+      - run: npm run build
+      - run: npm test
+  test-parser-js:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: 'parser/js'
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 'lts/Iron'
+          cache: 'npm'
+          cache-dependency-path: '**/**/package-lock.json'
+      - run: npm ci
+      - run: npm run build
+      - run: npm test
```

Comment:
Currently failing on the `feature/specification` branch (after switching to it). Can take a closer look at these next week if that would be helpful.

Errors:
<details>
FAIL test/unit/decoder/decodingUtils.spec.ts
  ● decode › decodeRle › should decode literals and runs in combination

    expect(received).toEqual(expected) // deep equality

    - Expected  - 49
    + Received  + 49

FAIL test/unit/evaluation/decodingUtils.spec.ts
  ● decode › decodeRle › should decode literals and runs in combination

    expect(received).toEqual(expected) // deep equality

    - Expected  - 49
    + Received  + 49
     ]

      93 |             const [values, newOffset] = decodeRle(encodedRleValues, numValues, true, 0);
      94 |
    > 95 |             expect(convertBigInt64ArrayToNumberArray(values)).toEqual([
         |                                                               ^
      96 |                 ...run1Values,
      97 |                 ...literalsValues,
      98 |                 ...run2Values,

      at Object.<anonymous> (test/unit/evaluation/decodingUtils.spec.ts:95:63)

FAIL test/unit/decoder/covtDecoder.spec.ts (1153.211 s)
  ● Console

    console.info
      data/amazon/5_5_11.covt

      at Object.<anonymous> (test/unit/decoder/covtDecoder.spec.ts:15:21)

    console.info
      data/bing/4-8-5.covt

      at Object.<anonymous> (test/unit/decoder/covtDecoder.spec.ts:30:21)

    console.info
      data/bing/5-16-11.covt

      at Object.<anonymous> (test/unit/decoder/covtDecoder.spec.ts:30:21)

  ● CovtDecoder › should decode Amazon based tiles

    Expected varint not more than 10 bytes

      159 |     if (b < 0x80) return [toNum(low, high), 10];
      160 |
    > 161 |     throw new Error("Expected varint not more than 10 bytes");
          |           ^
      162 | }
      163 |
      164 | function toNum(low, high) {

      at decodeVarintRemainder (src/decoder/decodingUtils.ts:161:11)
      at decodeVarint (src/decoder/decodingUtils.ts:136:12)
      at CovtDecoder.decodeLayerHeader (src/decoder/covtDecoder.ts:135:73)
      at new CovtDecoder (src/decoder/covtDecoder.ts:37:69)
      at Object.<anonymous> (test/unit/decoder/covtDecoder.spec.ts:20:33)
 ● CovtDecoder › should decode Bing Map based tiles

    expect(received).toEqual(expected) // deep equality

    - Expected  - 1729042
    + Received  +       0
          Objnode:internal/process/promises:289
            triggerUncaughtException(err, true /* fromPromise */);
            ^

[UnhandledPromiseRejection: This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason "SQLITE_CANTOPEN: unable to open database file".] {
  code: 'ERR_UNHANDLED_REJECTION'
}

Node.js v20.13.1

</details>